src/pages/AddLiquidity/index.tsx

```diff
@@ -596,7 +596,9 @@ export default function AddLiquidity({
                       <Trans>
                         Your liquidity will only earn fees when the market price of the pair is within your range.{' '}
                         <ExternalLink
-                          href={'https://docs.uniswap.org/concepts/introduction/liquidity-user-guide#4-set-price-range'}
+                          href={
+                            'https://docs.uniswap.org/protocol/concepts/introduction/liquidity-user-guide#4-set-price-range'
```

Comment:
Line break here as leaving it all on one line was giving me the eslinting prettier/prettier error